testsuite/MDAnalysisTests/core/test_universe.py

```diff
@@ -22,6 +22,10 @@
 from six.moves import cPickle
 
 import os
+try:
```

Comment:
Here you can use the six module instead of the a try except: `from six import StringIO`.